homeassistant/components/airly/strings.json

```diff
@@ -30,37 +30,37 @@
   "entity": {
     "sensor": {
       "caqi": {
-        "name": "CAQI"
+        "name": "Common air quality index"
```

Comment:
I used a name similar to AQI
https://github.com/home-assistant/core/blob/580b20b0a83c561986e7571b83df4a4bcb158392/homeassistant/components/sensor/strings.json#L131